x-pack/plugins/security_solution/public/timelines/containers/details/index.tsx

```diff
@@ -56,7 +56,9 @@ export const useTimelineEventsDetails = ({
   const refetch = useRef<() => Promise<void>>(asyncNoop);
   const abortCtrl = useRef(new AbortController());
   const searchSubscription$ = useRef(new Subscription());
-  const [loading, setLoading] = useState(false);
+
+  // loading = false initial state causes flashes of empty tables
+  const [loading, setLoading] = useState(true);
```

Comment:
why does purely cosmetic onboarding elements being added cause tables to flash empty?

Reply:
it was already flashing, it just became obvious with the addition of the EuiTourStep. if it starts as not loading, then loading, the element is there briefly and then disappears. EuiTourStep uses EuiPortal to mount. So it mounts to the flyout when it shows briefly, hides when the flyout flashes blank and goes into loading state, and comes back when flyout is actually available